Description: Bring Nag_Driver_Kolab::getChildren API up-to-date
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

Index: horde/nag/lib/Driver/Kolab.php
===================================================================
--- horde.orig/nag/lib/Driver/Kolab.php
+++ horde/nag/lib/Driver/Kolab.php
@@ -522,7 +522,7 @@ class Nag_Driver_Kolab extends Nag_Drive
      *
      * @return array  List of sub-tasks.
      */
-    public function getChildren($parentId)
+    public function getChildren($parentId, $include_history=true)
     {
         $task_list = $this->_getData()->getObjects();
         if (empty($task_list)) {
@@ -536,7 +536,7 @@ class Nag_Driver_Kolab extends Nag_Drive
                 continue;
             }
             $t = new Nag_Task($this, $this->_buildTask($task));
-            $children = $this->getChildren($t->id);
+            $children = $this->getChildren($t->id, $include_history);
             $t->mergeChildren($children);
             $tasks[] = $t;
         }
